### PR TITLE
UI Refinement for "Browse Now" option

### PR DESCRIPTION
**Pull request by foxtrotzulu94**
Changes include full view of the Pokemon's Information in DetailedPokemonActivity as well as improvements to the PokedexActivity. Everything is very loosely coupled, with the PokedexManager having most of the context sensitive information being fetched to update UI elements. There is extensive documentation and only small changes would be required once a fully working backend is finished.

### CODE REVIEW

File: app/src/main/java/me/quadphase/qpdex/DetailedPokemonActivity.java [hunk 1]
```diff
@@ -31,39 +141,453 @@
...
+                    ViewGroup.LayoutParams.MATCH_PARENT,
+                    ViewGroup.LayoutParams.MATCH_PARENT));
+            typeBadge.setScaleType(ImageView.ScaleType.FIT_XY);
+//            typeBadge.setMaxHeight(30);
+            typeBadge.setAdjustViewBounds(true);
+            typeBadge.getLayoutParams().width = 160;
+            frame.addView(typeBadge);
+        }
+
+        //TODO: We need to calculate for screen sizes
```
Comment (minicole) [inline app/src/main/java/me/quadphase/qpdex/DetailedPokemonActivity.java:246]: We should also consider what our app will look like on tablets. Android specifies that if you want to put an app in the AppStore for tablet, it must take advantage of the additional space, and not just be an extended version of the phone app.

File: app/src/main/java/me/quadphase/qpdex/PokedexActivity.java [hunk 2]
```diff
@@ -267,7 +295,7 @@ public void switchToPokemonData(View view){
         startActivity(intent);
     }
 
-    public void playPokemonCry(){
+    public void playPokemonCry(View view){
```
Comment (minicole) [inline app/src/main/java/me/quadphase/qpdex/PokedexActivity.java:298]: Why is View param passed here?

**Comment by minicole** [comment]
Your code looks good to me!
I just don't know about pushing all of the code from PokemonFactory that is not done yet. I guess you need the new methods to run your stuff though...

File: app/src/main/java/me/quadphase/qpdex/DetailedPokemonActivity.java (hunk 1 above)
Comment (foxtrotzulu94) [inline app/src/main/java/me/quadphase/qpdex/DetailedPokemonActivity.java:246]: Haven't tried building on a Tablet yet, but I know they have specific folders for that layout code. In this one the calculation is for correctly scaling the Type sprites into the GridView

File: app/src/main/java/me/quadphase/qpdex/PokedexActivity.java (hunk 2 above)
Comment (foxtrotzulu94) [inline app/src/main/java/me/quadphase/qpdex/PokedexActivity.java:298]: When you create an OnClick method, it asks for a View parameter if you want to link it from the XML. I left that possibility open here for now...

**Comment by foxtrotzulu94** [comment]
I actually don't call any of the PokemonFactory code here, just a lot of the Pokemon package code. Only the UI system was updated on this, though it plugs nicely into the backend stuff with PokedexManager. I've already merged this branch with issue-#12-dbAccess, so you'll see a lot of this code there (with slight modifications)

**Pull request merged**